src/completefunc.js

```diff
@@ -0,0 +1,21 @@
+import Icon2 from './more_vert.svg';
+
```

Comment:
- [ ] Great job working on this milestone. Adding a [live demo link (deployment) ](https://fdezcaminero.github.io/to-do-list/)is not a requirement for this milestone. Nevertheless, you have addded one, but it redirects to your repo. Kindly get the correct link or you can remove the link and replace with the text `coming soon....`